qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -240,6 +242,10 @@ def __init__(self, canvas, num):
         self.fit_browser.closing.connect(self.handle_fit_browser_close)
         self.window.setCentralWidget(canvas)
         self.window.addDockWidget(Qt.LeftDockWidgetArea, self.fit_browser)
+        # Need this line to stop the bug where the dock window snaps back to its original size after resizing.
+        # This is a qt bug reported at (https://bugreports.qt.io/browse/QTBUG-65592)
+        if QT_VERSION >= LooseVersion("5.6"):
+            self.window.resizeDocks([self.fit_browser], [40], Qt.Horizontal)
```

Comment:
Can we use keywords in this call to clarify the purpose of the arguments? The 40 is a little bit of a blackbox from the outside.